sdks/java/core/src/main/java/org/apache/beam/sdk/testing/UsesPythonExpansionService.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.testing;
+
+import org.apache.beam.sdk.annotations.Internal;
+
+/**
+ * Category tag for tests which use the expansion service in Python. Tests tagged with {@link
+ * UsesPythonExpansionService} should be run for runners which support cross-language transforms.
```

Comment:
Is that different from runners that can run tests tagged with UsesCrossLanguageTransforms ?

Reply:
The tests tagged with `UsesCrossLanguageTransforms` are picked up by all cross-language test setups (Java using Python, Python using Java) so it means that the tests should only use universal cross-language transform urns supported by all expansion services (Java expansion service, Python expansion service).

The tests tagged with `UsesPythonExpansionService` only use Python cross-language transforms so Java expansion service doesn't have to support the urn specified in the tests.

We could remove `UsesCrossLanguageTransforms` and split it into `UsesPythonExpansionService` and `UsesJavaExpansionService` in the future.

Reply:
+1